docs/CONTRIBUTING.md

```diff
@@ -33,6 +33,9 @@ Some helpful guides:
 - Redis for MacOS - need instructions
   An easier solution would be to use Docker.
 
+**SAML Setup**
+- Run `bash ./generate_ssl_certs.sh` in terminal
+
```

Comment:
I don't see the documentation update that you mention in the description, but someone is already working on this so don't worry about it. See #398